packages/internals/support/src/entities/variant.js

```diff
@@ -12,18 +12,34 @@ const parser = new Parser5({locationInfo: true});
 const _templates = new WeakMap();
 const _componentId = new WeakMap();
 
+const reservedConfigProps = [
+  'opts',
+  'files',
+  'views',
+  'previews',
+  'scenarios',
+  'templates'
```

Comment:
possibly also `component`? Or is this alterable?

Reply:
yup good point, will add it in